luigi-tutorials/tutorial-container.md

```diff
@@ -0,0 +1,294 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, programming-tool>javascript]
+primary_tag: topic>user-interface
+---
+
+# Implement Micro Frontends in Your UI5 App with Luigi Container
+<!-- description --> Learn how to build a UI5 application that utilizes the Luigi micro frontend framework. With the help of Luigi Container, you can create more modular and reusable applications in a simple way.
+
+## Prerequisites
+ - It is recommended to try the simpler Luigi examples on [GitHub](https://github.com/SAP/luigi/tree/master/core/examples) first to get acquainted with the framework.
+ - You need to have [Node.js](https://nodejs.org/en/download/current/) installed.
+
+## You will learn
+  - The basics of the Luigi micro frontend framework and Luigi Container 
+  - How to install Luigi Container
+  - How to use Luigi Container inside a sample UI5 application
+
+## Intro
+
+[Project "Luigi"](luigi-project.io) is an open-source micro frontend framework suitable for SAP environments, providing Fiori-compliant navigation out-of-the-box. Luigi is technology-agnostic, allowing you to create your app using any given frontend toolkit. 
+
+Normally, Luigi consists of two main parts: **Luigi Core** and **Luigi Client**. Core refers to the main app which houses the micro frontends, while Client refers to the micro frontends themselves. 
+
+However, a third feature called **Luigi Container** allows you to easily insert Luigi micro frontends anywhere without the need for a Luigi Core application. This feature simplifies the use of Luigi for developers, eliminating the need for significant changes in their applications.
+
+This tutorial will show you how to incorporate micro frontends inside a [UI5](https://sapui5.hana.ondemand.com/sdk/#/) application. Nevertheless, a similar process will also apply to other frontend frameworks such as [Angular](https://angular.io/) or [React](https://react.dev/).
+
+> **NOTE:** In addition to this tutorial, you can also experiment with the Luigi Container test app on [GitHub](https://github.com/SAP/luigi/tree/main/container/test-app).
+
+---
+
+## Steps
+
+### Install UI5 generator 
+
+1. Open a new command prompt/Terminal window and install the UI5 generator: 
+
+`npm install -g yo generator-easy-ui5`
+
+2. Run this command to ensure that Yeoman has been installed correctly: 
+
+`yo`
+
+Make sure the easy-ui5 generator is listed.
+
+### Create UI5 application 
+
+1. Create your UI5 project:
+
+`yo easy-ui5`
+
+2. Answer the prompts in this way to create your new project: 
+
+    <!-- border -->![UI5 Generator](ui5.png)
+
+3. Run the app locally:
+
+```shell
+cd luigi.ui5app
+npm start # or "yarn start"
+```
+
+### Install Luigi Container 
+
+Luigi Container can be installed via a [npm](https://www.npmjs.com/) package.
+
+To use npm packages in UI5, you need to first install the UI5 Tooling.
+
+1. In your project directory, run: 
+
+```shell
+npm i ui5-tooling-modules -D
```

Comment:
Stay consistent in the commands you use - as you used `npm install` above, please also use `npm install` here =>

`npm install ui5-tooling-modules -D`